db/sqitch.plan

```diff
@@ -31,5 +31,6 @@ mutations/create_application 2022-08-31T16:16:23Z Anthony Bushara <anthony@butto
 @1.0.0 2022-09-01T23:42:33Z Matthieu Foucault <matthieu@button.is> # release v1.0.0
 @1.0.1 2022-09-02T00:40:17Z Matthieu Foucault <matthieu@button.is> # release v1.0.1
 @1.1.0 2022-09-06T23:59:22Z Matthieu Foucault <matthieu@button.is> # release v1.1.0
-@1.1.1 2022-09-09T20:35:47Z Marcel Mueller,,, <marcel@Thinkpad> # release v1.1.1
+@1.1.1 2022-09-09T20:35:47Z Marcel Mueller <marcel@button.is> # release v1.1.1
```

Comment:
This line has been deployed to prod and cannot be updated, otherwise the change will have a different id, leading to the deploy failing.